### PR TITLE
Update GettingStarted.rst

### PR DESCRIPTION
**Pull request by dralabeing**
Corrected the path name to the design make file to use config.mk under Designs

### CODE REVIEW

**Review by maliberty — APPROVED** [review]

File: docs/user/GettingStarted.rst [hunk 1]
```diff
@@ -79,8 +79,8 @@ You can select a design using either of the following methods:
 1. The flow `Makefile`_ contains a list of sample design configurations at
 the top of the file.  Uncomment the respective line to select the design
 2. Specify the design using the shell environment, e.g.
-``make DESIGN_CONFIG=./designs/nangate45/swerv.mk`` or
-``export DESIGN_CONFIG=./designs/nangate45/swerv.mk ; make``
+``make DESIGN_CONFIG=./designs/nangate45/swerve/config.mk`` or
```
Comment (rovinski) [inline docs/user/GettingStarted.rst:82]: It's "swerv" not "swerve"

**Review by rovinski — CHANGES_REQUESTED** [review]

**Comment by dralabeing** [comment]
Thanks...will fix

**Comment by dralabeing** [comment]
I just created a new pull request with this fix- is there a way to modify
and use original pull request? Just checking on the correct procedure...
thanks

**Comment by maliberty** [comment]
If you just push to the same branch the PR will update.

**Comment by dralabeing** [comment]
Thanks Matt...fixed

**Comment by rovinski** [comment]
@dralabeing you can see at the top of this PR that it was opened from branch `patch-2`. You need to push to that specific branch to update this PR, not create a new branch.

File: docs/user/GettingStarted.rst (hunk 1 above)
Comment (dralabeing) [reply docs/user/GettingStarted.rst]: Fixed swerve to swerv

**Comment by dralabeing** [comment]
Thanks  Austin.. new to GitHub.. I think I got it right this time

**Pull request merged**